app/service/form_service.rb

```diff
@@ -0,0 +1,14 @@
+class FormService
+  include Rails.application.routes.url_helpers
+
+  def initialize(form)
+    @form = form
+  end
+
+  def path_to
```

Comment:
Wondering if this method name makes sense. Just thinking about what it does and if something like `path_for_state` or `state_path` or something...

Reply:
yeh, happy to change to `path_for_state`

Reply:
updated